modules/ROOT/pages/errors/gql-errors/42I13.adoc

```diff
@@ -1,4 +1,5 @@
 = 42I13
+label:new[Changed in 2025.03]
```

Comment:
@Lojjs hi! Is this code really changed in 2025.03? You mentioned it on the trello card, and in the spreadsheet it has a comment 'message changed in 2025.03'. But if you take a look at https://github.com/neo-technology/neo4j/pull/29666/files#diff-384932958624a4a10f07c7751fd5d9840d7a721fcc8e01d0b7e93aa1711d87c7, there is no diff. Formatting is different, but words are the same

Reply:
label 'Changed' is removed